project/AppDependencies.scala

```diff
@@ -1,21 +1,17 @@
 import play.core.PlayVersion
 import play.sbt.PlayImport.ws
 import sbt._
+
 object AppDependencies {
   val compile = Seq(
     ws,
-    "uk.gov.hmrc" %% "simple-reactivemongo" % "7.27.0-play-26",
-    "uk.gov.hmrc" %% "bootstrap-play-26" % "1.8.0",
-    "uk.gov.hmrc" %% "order-id-encoder" % "1.1.0",
-    "com.beachape" %% "enumeratum" % "1.6.0",
     "uk.gov.hmrc" %% "auth-client" % "3.0.0-play-26",
```

Comment:
we don't need to specify auth-client 3.0.0 to build and test the project but we do to keep bobby happy it would seem.

I'll probably raise a few PRs to add this back in to a few projects where I removed it.